The instructions to build fundamental.hxx were always there, but for
some reason dports gmake did not feel the need to build it.  Only
by adding it as a fundamental.hxx dependency did it get built.

--- libxsd-frontend/xsd-frontend/makefile.orig	2010-04-27 19:31:24.000000000 +0000
+++ libxsd-frontend/xsd-frontend/makefile
@@ -52,6 +52,8 @@ cxx_od  := $(cxx_obj:.o=.o.d)
 m4_cxx  := $(out_base)/semantic-graph/fundamental.hxx \
   $(out_base)/semantic-graph/fundamental.cxx
 
+$(out_base)/semantic-graph/fundamental.cxx: $(out_base)/semantic-graph/fundamental.hxx
+
 xsd_frontend.l             := $(out_base)/xsd-frontend.l
 xsd_frontend.l.cpp-options := $(out_base)/xsd-frontend.l.cpp-options
 
